scripts/mysql_slave_lag.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+
+mysql_status=$(mysql -s -e 'show slave status\G')
+seconds=$(echo "$mysql_status" | grep Seconds_Behind_Master | awk {'print $2'})
+running=$(echo "$mysql_status" | grep Slave_SQL_Running | awk {'print $2'})
+
+test "$running" = "Yes"
+retval=$?
+if [ $retval -ne 0 ]; then
```

Comment:
Lines 8-10 look redundant.  Could just say if [ "$running" = "Yes" ];   Also makes it more readable to see what you're checking.